lib/goo/sparql/mixins/solution_lang_filter.rb

```diff
@@ -141,6 +144,10 @@ def fill_other_languages?
           @fill_other_languages
         end
 
+        def is_literal(object)

```

Comment:
in ruby, we write `literal?`

Reply:
5f81e7b624af2b1bbb09f9434ff86111862fb4bc